runtime/near-vm-runner/fuzz/fuzz_targets/diffrunner.rs

```diff
@@ -19,20 +19,22 @@ fn run_fuzz(code: &ContractCode, vm_kind: VMKind) -> VMOutcome {
     let mut fake_external = MockedExternal::new();
     let mut context = create_context(vec![]);
     context.prepaid_gas = 10u64.pow(14);
-    let mut config = Config::test();
-    config.limit_config.contract_prepare_version =
+    let config_store = RuntimeConfigStore::new(None);
+    let config = config_store.get_config(!0);
```

Comment:
I don't really see it as a pain point… There isn't a great place for an abstracting function either, since tests are all over the place, and it doesn't fit into `RuntimeConfigStore` itself either…